### ISSUE TITLE
the website seems down, cheat gem installed under 1.9.3 can not retrieve the page

### ISSUE THREAD

**Issue by ipstone** [defunkt/cheat#14]

**Comment by MGPalmer** [comment]
Yep. Lots of dead links on http://cheat.errtheblog.com/ as well. Is this project dead ?

**Comment by sferik** [comment]
Cheat is currently operating in read-only mode, meaning you can read existing cheat sheets but not edit them or create new ones. The project is not dead. It's maintained by volunteers. The original code broke a few months ago and needed to be rewritten from scratch. The new maintainers (myself and @amateurhuman) have reimplemented the read-only functionality first and plan to implement additional functionality over time, but there is no set schedule. If you would like to accelerate the rate at which functionality is restored, we would welcome your assistance.

**Comment by MGPalmer** [comment]
@sferik Well crap now I feel like a dick :( I didn't want to offend, sorry. Let me offer some more details:

```
$ gem install cheat
Successfully installed cheat-1.3.3

$ cheat sheets
/home/mt/.rvm/rubies/ruby-1.9.3-p194/lib/ruby/1.9.1/open-uri.rb:346:in `open_http': 404 Not Found (OpenURI::HTTPError)

$ cheat sprinkle
Whoa, some kind of Internets error!
=> 404 Not Found from http://cheat.errtheblog.com:80/y/sprinkle
```

I notice that other sheets are available locally if you know their names.
Is this part of the current overhaul and can't work right now or is this a new bug ?

Thanks for the effort BTW !

**Comment by localhots** [comment]
Any chances to make it work again? I really loved this tool.

**Comment by metasoarous** [comment]
SOOOO SADDDDDDDD!!!

I would love to get this back up and working fully again. Please let me know if there is any way that I can help.

**Comment by bolpin** [comment]
It is working in read-only mode for me.  

Would it be possible to post a list of sheets available until the 'cheat sheets' command is back up?

I'd be happy to help on this.  Thanks!

**Comment by skull-squadron** [comment]
Looks like all the sheets are missing, perhaps it's just the github.io domain change.

Might it make sense to make the sheets a github repo and served on github.io, fewer moving parts.

**Comment by jrhorn424** [comment]
@sferik I had a look at your ginourmous list of repos, and couldn't find cheat. Is the source hosted somewhere else?

**Comment by ipstone** [comment]
guess defunkt is too busy at the moment ... 
just fyi, there's a python tool https://github.com/chrisallenlane/cheat 

I still think ruby version may become more useful/expressive, we need some efforts here

I would prefer not to use server/client mode, just a directory of sheets, and ability for the 'cheat' program to 'learn' from clipboard, or an url (do some formatting before manual edit) ...
